boards/arm/stm32h747i_disco/Kconfig.defconfig

```diff
@@ -38,28 +38,21 @@ config CLOCK_STM32_D3PPRE
 config STM32H7_DUAL_CORE
 	default y
 
-# Dual core boot configuration
-if STM32H7_DUAL_CORE
 # Boot method is selected in common file to avoid desync issues
 choice STM32H7_DUAL_CORE_BOOT
 	# Use out of the box config by default
 	# default STM32H7_BOOT_CM4_CM7
 	default STM32H7_BOOT_CM7_CM4GATED
+	# Dual core boot configuration
```

Comment:
Please don't move the comment

Reply:
Moved back. Thought it was about the symbol in the `if`.